tests/integration/test_user_invite.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from unittest.mock import patch
+from models.user import RoleEnum
```

Comment:
`RoleEnum` isn't used here, so we can delete this import.

Reply:
Thanks, I will remove this.